.github/workflows/ci.yml

```diff
@@ -24,3 +24,15 @@ jobs:
 
       - name: Run tests
         run: npm test
+
+      - name: Install Pandoc
+        run: sudo apt-get install -y pandoc
+
+      - name: Check generated README is up to date
+        run: |
+          ./generate
+          if [ ! -z "$(git status --porcelain ../README.md)" ]; then
+            echo "The README generated by `docs/generate` has changed! If you edited the README directly, please edit the files in `docs` instead and regenerate the README. If you modified the files in `docs`, regenerate the README. To do so, cd into `docs` and run `./generate`."
```

Comment:
I think this error message can be improved. Do you have any suggestions?

Reply:
I think it's great! The only way it could be improved would be to harden the `docs/generate` script such that we can call it from any directory. Then we can skip the `working-directory: docs` part in the YAML file and the "cd into…" part in the error message.